docs/man/sat-bootsys.8.rst

```diff
@@ -309,7 +309,7 @@ These options set the timeouts of various parts of the stages of the
 **--bos-shutdown-timeout** *BOS_SHUTDOWN_TIMEOUT*
         Timeout, in seconds, to wait until compute and
         application nodes have completed their BOS shutdown.
-        Defaults to 600. Overrides the option
+        Defaults to -1 for an infinite timeout. Overrides the option
```

Comment:
seems like this got skipped to be updated

Reply:
Ohh, Thanks